docker-compose/local/setup.sh

```diff
@@ -287,18 +287,30 @@ generate_key() {
 
 echo $(show_message "security_secrect_regenerate")
 
+# check operating system
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    # macOS
+    SED_COMMAND="sed -i ''"
+elif [[ "$OSTYPE" == "linux-gnu"* ]]; then
+    # Linux
+    SED_COMMAND="sed -i"
+else
+    echo "Unsupported OS"
```

Comment:
In the original design, the action to regenerate secrets is optional, we should not exit here.

Reply:
sure